app/bundles/LeadBundle/Translations/en_US/messages.ini

```diff
@@ -476,4 +476,6 @@ mautic.lead.field.companywebsite="Website"
 mautic.lead.field.companynumber_of_employees="Number of Employees"
 mautic.lead.field.companyfax="Fax"
 mautic.lead.field.companyannual_revenue="Annual Revenue"
-mautic.lead.company.field.owner="Company Owner"
\ No newline at end of file
+mautic.lead.company.field.owner="Company Owner"
+mautic.lead.lead.events.changecompanyscore="Change company score"
```

Comment:
We call this `score` for companies but `points` for contacts. Will that lead to confusion?